src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -653,20 +653,22 @@ class TezosNodeOperator(
     val proposalsStateFetch =
       fetchMerge(currentQuorumFetcher, currentProposalFetcher)(CurrentVotes.apply)
 
-    def parseMichelsonScripts: Block => Block = {
+    def parseMichelsonScripts: Block => Block = { block: Block =>
```

Comment:
Why re-introduce the parameter to create an explicit lambda on top of an existing function?

Reply:
because I needed explicit block to extract micheline from the lens before parsing, maybe I misses simpler solution, do you have any suggestions?

Reply:
no, I simply missed this detail, thanks!